extensions/poetry/main.ts

```diff
@@ -42,6 +42,12 @@ async function poetryCheckDryRun(subcommand: string, args: string[]): number {
   await lockfileBackup.backup()
   await manifestBackup.backup()
 
+  if (manifestBackup.fileContent === null) {
+    console.log(`\`pyproject.toml\` was not found in the current directory.`)
+    console.log(`Please move to the Poetry project's top level directory and try again.`)
+    return 127
+  }
```

Comment:
Can we not check for these files to exist **before** trying to back them up? This seems like a strange order of operations.

Or are you trying to also cover issues like permission errors? In that case the error message isn't clear.

Since this is an error, I'd probably also use `console.error` or `console.warn`, right? To get it to STDERR maybe?